test/test_convert.c

```diff
@@ -0,0 +1,100 @@
+#include <unity.h>
+#include "simulation_lifecycle/convert.h"
+#include "simulation_lifecycle/error.h"
+
+void setUp(void) {
+    // set stuff up here
+}
+
+void tearDown(void) {
+    // clean stuff up here
+}
+
+/**
+ *  Function convert_results - Test case #1
+ *  Description: Call the function with path being null.
+ *  Expected output: The function returns 340.
+ */
+void test_function_convert_results_test_case_1(void) {
+    char *results_folder_path = NULL;
+    int error_code = convert_results(results_folder_path);
+    TEST_ASSERT_EQUAL(error_code, INPUT_PATH_EMPTY);

```

Comment:
normally, you use `TEST_ASSERT_EQUAL(expected, actual)`.
It is not a big deal, but I suggest you change this to `TEST_ASSERT_EQUAL(INPUT_PATH_EMPTY, error_code)`.
The same for the other tests

Reply:
Changes have been made in the new commit :) Thanks!